save-demo/src/main/kotlin/com/saveourtool/save/demo/utils/HttpUtils.kt

```diff
@@ -21,6 +21,12 @@ private const val DEFAULT_BUFFER_SIZE = 4096
 private const val PORT_RANGE_FROM = 20000
 private const val PORT_RANGE_TO = 30000
 
+/**
+ * Subdomain of all save-demo-agents
+ */
+const val DEMO_SUBDOMAIN_NAME = "demo-agent-service"
+const val CLUSTER_DOMAIN = "cluster.local"
+
 private val logger = LoggerFactory.getLogger("c.s.s.d.u.HttpUtils")
```

Comment:
maybe change to `LoggerFactory.getLogger(object {}.javaClass.enclosingClass::class.java)`?

Reply:
Ok, will do it